assets/sass/components/global/_googlesitekit-publisher-wins.scss

```diff
@@ -84,6 +84,10 @@
 			display: inline-block;
 			max-width: 100%;
 		}
+
+		svg {
+			width: 200px;
+		}
```

Comment:
Looks like we might need to add some responsive styles here or make it relative so that it naturally fills the grid cell width? I'm guessing the previous images would have scaled as well.

Reply:
This is purely for the rocket image which was 200px x 167px, and never scaled, see the comparison images. Further confirmation is that the addition of the rule brings no additional VRT test failures, so if this rule has an unintended side effect, it also means we are missing some storybook tests.

SVG1
https://github.com/google/site-kit-wp/blob/7dc1eb3947860fb77a9e98b2fcccdebca1248c91/tests/backstop/reference/google-site-kit_Global_Notifications_Module_Setup_Complete_0_document_0_small.png

PNG1
https://github.com/google/site-kit-wp/blob/develop/tests/backstop/reference/google-site-kit_Global_Notifications_Module_Setup_Complete_0_document_0_small.png

SVG2
https://github.com/google/site-kit-wp/blob/7dc1eb3947860fb77a9e98b2fcccdebca1248c91/tests/backstop/reference/google-site-kit_Global_Notifications_Module_Setup_Complete_0_document_1_medium.png

PNG2
https://github.com/google/site-kit-wp/blob/develop/tests/backstop/reference/google-site-kit_Global_Notifications_Module_Setup_Complete_0_document_1_medium.png

SVG3
https://github.com/google/site-kit-wp/blob/7dc1eb3947860fb77a9e98b2fcccdebca1248c91/tests/backstop/reference/google-site-kit_Global_Notifications_Module_Setup_Complete_0_document_2_large.png

PNG3
https://github.com/google/site-kit-wp/blob/develop/tests/backstop/reference/google-site-kit_Global_Notifications_Module_Setup_Complete_0_document_2_large.png

Reply:
@aaemnnosttv updated the rule in https://github.com/google/site-kit-wp/pull/2834/commits/6f76bed4afb4e3870f9b2ad0fd54e74335c231b4

Updated the image in https://github.com/google/site-kit-wp/pull/2834/commits/cbeb8a16b4f4ebc10f2b88ebd09eef07fa072bef